### PR TITLE
Show/hide call view controls in remote video mode if user taps screen.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk

### CODE REVIEW

File: Signal/src/views/OWSAnyTouchGestureRecognizer.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#import <UIKit/UIKit.h>
+
+// This custom GR can be used to detect touches when they
+// begin in a view.  In order to honor touch dispatch, this
+// GR will ignore touches that:
+//
+// * Are not single touches.
+// * Are not in the view for this GR.
+// * Are inside a visible, interaction-enabled subview.
+@interface OWSAnyTouchGestureRecognizer : UIGestureRecognizer
```
Comment (charlesmchen) [inline Signal/src/views/OWSAnyTouchGestureRecognizer.h]: We use a custom GR in order to be more responsive, specifically to recognize as soon as the user taps the screen, not when a "tap" is completed.

Comment (charlesmchen) [inline Signal/src/views/OWSAnyTouchGestureRecognizer.h]: I'd like to use this kind of GR for certain other interactions in the UI as well.

Comment (michaelkirk) [inline Signal/src/views/OWSAnyTouchGestureRecognizer.h]: I like how responsive it is. 👍

Comment (charlesmchen) [inline Signal/src/views/OWSAnyTouchGestureRecognizer.h]: @michaelkirk  Is this work LGTM?

Comment (michaelkirk) [inline Signal/src/views/OWSAnyTouchGestureRecognizer.h]: Yup, LGTM here and #1656

**Pull request merged**